src/vs/workbench/contrib/debug/browser/variablesView.ts

```diff
@@ -123,9 +123,10 @@ export class VariablesView extends ViewPane {
 		container.classList.add('debug-variables');
 		const treeContainer = renderViewTree(container);
 		const linkDetector = this.instantiationService.createInstance(LinkDetector);
+		const displayType: boolean = this.configurationService.getValue('variableDisplayType.variableView');
```

Comment:
This isn't registered as a configuration (in `debug.contribution.ts`) so users cannot find it. Also, it should be prefixed with `debug.` like other debug configuration keys, e.g. `debug.showVariableTypes` would work.

A single config is probably fine for both the variables/watch view.